plothist/plotters.py

```diff
@@ -63,6 +89,36 @@ def create_variable_registry(variables, path="./variable_registry.yaml", reset=F
 
 
 def get_variable_from_registry(variable, path="./variable_registry.yaml"):
+    """
+    This function retrieves the parameter information for a variable from the variable registry file specified by the 'path' parameter.
+    It loads the variable registry file and returns the dictionary entry corresponding to the specified variable name.
+
+    If the variable registry file does not exist, a `RuntimeError` is raised, unless the default path ("./variable_registry.yaml") is used.
+    In that case, it is assumed that the `create_variable_registry` function has not been executed yet.
+
+
+    Parameters
+    ----------
+    variable : str
+        The name of the variable for which to retrieve parameter information.
+    path : str, optional
+        The path to the variable registry file (default is "./variable_registry.yaml").
+
+    Returns
+    -------
+    dict
+        A dictionary containing the parameter information for the specified variable.
+
+    Raises
+    ------
+    RuntimeError
+        If the variable registry file does not exist, and the default path is used.
+
+    See also
+    --------
+    create_variable_registry
+    """
+
     if not os.path.exists(path):
         if path == "./variable_registry.yaml":
```

Comment:
Weird behavior. We should always raise a RuntimeError if the registry does not exist

Reply:
[same](https://github.com/cyrraz/plothist/pull/3#discussion_r1220610562)